attributes/default.rb

```diff
@@ -464,7 +464,7 @@
 default['cluster']['volume'] = nil
 
 # ParallelCluster internal variables to configure active directory service
-default['cluster']["directory_service"]["enabled"] = nil
+default['cluster']["directory_service"]["enabled"] = 'false'
```

Comment:
instead of changing the default value, would it make sense to instead change the value we use to decide whether or not to skip execution of the directory service recipe ([here](https://github.com/aws/aws-parallelcluster-cookbook/blob/wip/ad_integration/cookbooks/aws-parallelcluster-config/recipes/directory_service.rb#L20))?

Reply:
I believe this is better. This approach is compatible with CLI passing in the `false` explicitly

Reply:
okay. my comment was based on the idea that it would make sense to have only one value enable the feature as opposed to only one value disabling it. but either works.